### PR TITLE
reduce sync-status log

### LINKED ISSUE #6116: Reduce sync-status log
## Enhancement
Currently, TiFlash will print sync-status log when response sync-status request from TiDB. It's no need to print log if there is no lag regions.
e9c9c75d-d5b8-4ec0-a0c6-fef0367d1666

### PR DESCRIPTION
**Pull request by hehechen**
Signed-off-by: hehechen <awd123456sss@gmail.com>

### What problem does this PR solve?

Issue Number: close #6116

Problem Summary:
Reduce sync-status log.
### Check List

Tests 

- [x] Unit test
- [ ] Integration test
- [ ] Manual test (add detailed scripts or steps below)
- [ ] No code

Side effects

- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [ ] Breaking backward compatibility

Documentation

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Changes MySQL compatibility

### Release note

```release-note
None
```

### CODE REVIEW

File: dbms/src/Storages/Transaction/ProxyFFIStatusService.cpp [hunk 1]
```diff
@@ -81,7 +81,10 @@ HttpRequestRes HandleHttpRequestSyncStatus(
                 }
             }
             count = region_list.size();
-            LOG_DEBUG(&Poco::Logger::get(__FUNCTION__), "table_id={}, total_region_count={}, ready_region_count={}, lag_region_info={}", table_id, regions.size(), count, lag_regions_log.toString());
+            if (count < regions.size())
+            {
+                LOG_DEBUG(&Poco::Logger::get(__FUNCTION__), "table_id={}, total_region_count={}, ready_region_count={}, lag_region_info={}", table_id, regions.size(), count, lag_regions_log.toString());
+            }
```
Comment (Lloyd-Pottiger) [inline dbms/src/Storages/Transaction/ProxyFFIStatusService.cpp:87]:
```suggestion
                LOG_DEBUG(Logger::get(__FUNCTION__), "table_id={}, total_region_count={}, ready_region_count={}, lag_region_info={}", table_id, regions.size(), count, lag_regions_log.toString());
            }
```

**Review by Lloyd-Pottiger — APPROVED** [review]

File: dbms/src/Storages/Transaction/ProxyFFIStatusService.cpp [hunk 2]
```diff
@@ -81,7 +81,10 @@ HttpRequestRes HandleHttpRequestSyncStatus(
                 }
             }
             count = region_list.size();
```
Comment (JaySon-Huang) [inline dbms/src/Storages/Transaction/ProxyFFIStatusService.cpp:83]: Better rename "count" -> "ready_region_count", and use "=" instead of ":" in line 78

**Review by JaySon-Huang — APPROVED** [review]
LGTM with one comment

**Pull request merged**